python/src/numerous/collection/numerous_document.py

```diff
@@ -15,44 +15,57 @@ class NumerousDocument:
     Attributes
     ----------
         key (str): The key of the document.
-        collection_id (str): The id of collection document belongs to.
-        data (Optional[Dict[str, Any]]): The data of the document.
+        collection_info tuple[str, str]: The id
+        and key of collection document belongs to.
+        data (Optional[dict[str, Any]]): The data of the document.
```

Comment:
I think these lines should be indented to indicate that they are a continuation of the above